src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,4 @@
+package duke;
+
+public class DukeException extends Exception {
```

Comment:
You can think of exploiting the custom exception to give a reasonable custom message (or any other reasonable custom processing)